agent/util-scripts/gold/pbench-metadata-log/test-03.txt

```diff
@@ -68,33 +79,31 @@ turbostat: --interval=3
 
 [tools/dhcp31-174]
 hostname-s = dhcp31-174
-sar:
 
 [tools/perf104]
 hostname-s = perf104
-iostat:
 
 --- pbench tree state
 +++ test-execution.log file contents
 /var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/yum list installed pbench-agent
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/scp -r dhcp31-174:/var/tmp/pbench-test-utils/pbench/tools-default /var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/iostat:foo
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/mpstat:bar
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/perf:baz
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/pidstat:hunoz
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/proc-interrupts:hukerz
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/proc-vmstat:foobar
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/sar:foo
-/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/tools-default/sar:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog/dhcp31-174/turbostat:bar
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/scp -r dhcp31-174:/var/tmp/pbench-test-utils/pbench/tools-default /var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/iostat:foo
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/mpstat:bar
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/perf:baz
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/pidstat:hunoz
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/proc-interrupts:hukerz
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/proc-vmstat:foobar
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/sar:foo
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/turbostat:bar
```

Comment:
Why do we lose `.../dhcp31-174/tools-default/sar` directory munged on this line?  Seems like a good fix, but would love to know why this was fixed.

Reply:
That's a bug in the mocked scp. I'll post a fix on @riya-17's branch.